osm2streets/src/road.rs

```diff
@@ -239,6 +278,137 @@ impl Road {
         self.lane_specs_ltr.iter().map(|l| l.width).sum()
     }
 
+    /// Calculates the number of (forward, both_ways, backward) lanes. The order of the lanes
+    /// doesn't matter.
+    pub fn driving_lane_counts(&self) -> (usize, usize, usize) {
```

Comment:
`driving_lane` makes me think `LaneType::Driving`, but you're looking for the lanes suffix